File: java/src/Node.java
import java.io.Console;

/**
 * Created by Aarthi on 8/3/15.
 */
public class Node {
    public boolean Visited;
    public int Data;
    public Node Left;
    public Node Right;

    public Node(Node left, Node right, int data) {
        this.Data = data;
        this.Left = left;
        this.Right = right;
    }

    public static void Print(Node node) {
        System.out.println(node.Data);
    }

    public static void Visit(Node node){
        node.Visited = true;
    }
}

File: csharp/Node.cs
using System;

namespace Fall2015
{
	class Node {

		public bool Visited { get; private set;}
		public int Value { get; private set;}
		public Node Left { get; private set;}
		public Node Right {get;private set;}

		public Node(int value) {
			this.Value = value;
		}

		public static void Print(Node node) {
			Console.WriteLine(node.Value);
		}

		public static void Visit(Node node){
			node.Visited = true;
		}
	}
}
